server/deadfish.hpp

```diff
@@ -100,19 +100,44 @@ struct Civilian : public Mob {
 	void collisionResolution();
 };
 
+// just a data container to be able to send it later to clients
+struct Tileinfo {
+	Tileinfo(const FlatBuffGenerated::Tileinfo* fb_Ti) : name(fb_Ti->name()->str()), gid(fb_Ti->gid()) {}
+	std::string name;
+	uint16_t gid;
+};
+
+// just a data container to be able to send it later to clients
```

Comment:
Just one comment on the subject is probably informative enough :P sorry for being very nitpicky but dat shiet b messin w/ my ocd ni

Reply:
It shows up as the basic object documentation, like when you hover over the type name in IDE. In `deadfish.hpp` it may be OCD-inducing, but it's intentional as well :P